sklearn/decomposition/nmf.py

```diff
@@ -842,8 +842,13 @@ class NMF(BaseEstimator, TransformerMixin):
     solver : 'pg' | 'cd'
         Numerical solver to use:
         'pg' is a (deprecated) Projected Gradient solver.
+
+        .. versionchanged:: 0.17
```

Comment:
yes, only that I would make the "versionchanged" "Deprecated projected gradient solver".